Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -17,10 +17,26 @@ public struct Configuration: Hashable {
         case allEnabled
     }
 
+    public enum IndentationMode: Equatable {
```

Comment:
Suggestion: this seems like more of a "configuration" than a "mode. `IndentationConfiguration`

Reply:
Or given that it's nested within Configuration, just `Indentation` actually. `Configuration.Indentation`

Reply:
Agreed - that's much nicer